src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
 export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
it's better to create types for color and shape.
And whenever you get shape or color add created types. Check other cases

type Shapes = 'triangle' | 'circle' | 'rectangle';
type Colors = 'red' | 'green' | 'blue';


```suggestion
  shape: Shapes;
  color: Colors;
```